Cargo.toml

```diff
@@ -26,7 +27,7 @@ regex = "1.5.4"
 reqwest = { version = "0.11.3", features = ["json"] }
 rocket = { version = "0.5.0-rc.1", features = ["tls", "json"] }
 rocket_codegen = { version = "0.5.0-rc.1" }
-redis = { version = "0.21", features = ["r2d2"] }
+redis = { version = "0.21", features = ["tokio-comp"] }
```

Comment:
If `bb8-redis` handles the async pool for us do we still need `tokio-comp`?

Reply:
Just saw that they use this as a dependency on the `bb8` side:

https://github.com/djc/bb8/blob/227ee229351106b3435c00ca187a9a1a1b14185c/redis/Cargo.toml#L12

So I'd say that if we use `redis` directly then we should keep this (so that the version is pinned). If we don't then we can remove this dependency completely.